doc/usermanual.md

```diff
@@ -83,7 +83,7 @@ Returns the current log level, in a json payload with a single attribute `level`
 
 ### Transactions
 The library implements a concept of transactions, in order to follow the execution flow the library follows when treating
-requests entering both from the Northbound and the Southbound.
+requests entering both from the North and the South ports of the IoT Agent.
```

Comment:
In lines 521:

> This collection of utility middlewares is aimed to be used in the northbound of the IoTAgent Library

Reply:
Fixed a7e6db8 - Replace with *to the north of*